tests/phpunit/tests/rest-api/rest-search-controller.php

```diff
@@ -443,7 +443,6 @@ public function test_prepare_item_limit_fields() {
 			array(
 				'id',
 				'title',
-				'_links',
```

Comment:
Is this not indicative of a breaking change?

Reply:
Yes, I think that's fair to say. It's a reflection of the fact that, currently, when `_fields` is used and doesn't include `_links`, then `_links` is first included in the response data, then removed by `rest_filter_response_fields()`. Under this patch, `_links` wouldn't be in the response data at all.

Reply:
Understood. I think that is acceptable breakage. Let's make sure it gets documented in the dev note though.